src/amuse/support/state.py

```diff
@@ -139,11 +138,7 @@ def __init__(self, interface, **options):
     @option(type='boolean', sections=['state',])
     def is_enabled(self):
         return True
-        
-    @option(type='boolean', sections=['state',])
```

Comment:
why remove this? maybe just default to True instead?

Reply:
its just duplicating logging functionality..we also don't do it when logging the channel stuff...